torch/nn/_functions/thnn/activation.py

```diff
@@ -95,7 +98,47 @@ def backward(self, grad_output):
             self.lower,
             self.upper,
             self.train,
-            self.inplace
+            False
+        )
+        return grad_input
+
+
+class SELU(InplaceFunction):
+
+    def __init__(self, inplace=False):
+        super(SELU, self).__init__(inplace)
+        self.alpha = 1.6732632423543772848170429916717
+        self.scale = 1.0507009873554804934193349852946
```

Comment:
I'm not sure we want to allow the user to specify them. The reason why they chose these parameters is to enforce that the activations have on average zero mean and unit std.
I'll modify it to be new-style